src/utils.ts

```diff
@@ -35,8 +35,6 @@ import { W, WriteConcern, WriteConcernOptions } from './write_concern';
  * @public
  */
 export type Callback<T = any> = (error?: AnyError, result?: T) => void;
```

Comment:
If this type is exported, is this technically a breaking change?

Reply:
It was never used on any of our public APIs but yes, if we feel like users may have imported this type we should revert this. Should I deprecate it perhaps?